ci/slt/pipeline.yml

```diff
@@ -8,7 +8,8 @@
 # by the Apache License, Version 2.0.
 
 steps:
-  - command: ci/slt/slt.sh
```

Comment:
@benesch I am not 100% certain about this part. How can I trigger to see if buildkite is happy? Previously you found a way to run nightly against a commit from a personal Materialize fork, but I am not sure how you did it.

Reply:
Nevermind, I think I figured it out -- opening  a pull request seems to do something that causes the git sha from the fork to become available for checkout from the main Materialize repository ... so the "New Build" button in the Buildkite UI can be used to build that sha.

Reply:
It is green, so I guess it is all good

https://buildkite.com/materialize/sql-logic-tests/builds/4972#11242dcb-7eef-40c7-91d2-61fcaaabbb63

Reply:
Yep, that's exactly right!